docs/source/chs_test_params.csv

```diff
@@ -1,209 +1,209 @@
-loading,7,y5,fac2,-inf,inf,0.81524559,0.8,1
-loading,7,y6,fac2,-inf,inf,0.57084593,0.6,1
-loading,7,Q1,fac1,-inf,inf,0.93520167,0.9,1
-meas_sd,0,y1,-,-inf,inf,0.50497719,0.5,1
-meas_sd,0,y2,-,-inf,inf,0.50088168,0.5,1
-meas_sd,0,y3,-,-inf,inf,0.48136282,0.5,1
-meas_sd,0,y4,-,-inf,inf,0.53215346,0.5,1
-meas_sd,0,y5,-,-inf,inf,0.47039143,0.5,1
-meas_sd,0,y6,-,-inf,inf,0.48344469,0.5,1
-meas_sd,0,y7,-,-inf,inf,0.47943359,0.5,1
-meas_sd,0,y8,-,-inf,inf,0.53421227,0.5,1
-meas_sd,0,y9,-,-inf,inf,0.50146093,0.5,1
-meas_sd,1,y1,-,-inf,inf,0.49105567,0.5,1
-meas_sd,1,y2,-,-inf,inf,0.49870431,0.5,1
-meas_sd,1,y3,-,-inf,inf,0.50514084,0.5,1
-meas_sd,1,y4,-,-inf,inf,0.49743526,0.5,1
-meas_sd,1,y5,-,-inf,inf,0.4994178,0.5,1
-meas_sd,1,y6,-,-inf,inf,0.50424182,0.5,1
-meas_sd,2,y1,-,-inf,inf,0.50427244,0.5,1
-meas_sd,2,y2,-,-inf,inf,0.51856939,0.5,1
-meas_sd,2,y3,-,-inf,inf,0.50392617,0.5,1
-meas_sd,2,y4,-,-inf,inf,0.49161026,0.5,1
-meas_sd,2,y5,-,-inf,inf,0.50441808,0.5,1
-meas_sd,2,y6,-,-inf,inf,0.48482939,0.5,1
-meas_sd,3,y1,-,-inf,inf,0.49476345,0.5,1
-meas_sd,3,y2,-,-inf,inf,0.49363682,0.5,1
-meas_sd,3,y3,-,-inf,inf,0.49918763,0.5,1
-meas_sd,3,y4,-,-inf,inf,0.49728617,0.5,1
-meas_sd,3,y5,-,-inf,inf,0.5116465,0.5,1
-meas_sd,3,y6,-,-inf,inf,0.48035036,0.5,1
-meas_sd,4,y1,-,-inf,inf,0.50529312,0.5,1
-meas_sd,4,y2,-,-inf,inf,0.50706948,0.5,1
-meas_sd,4,y3,-,-inf,inf,0.47849704,0.5,1
-meas_sd,4,y4,-,-inf,inf,0.49962829,0.5,1
-meas_sd,4,y5,-,-inf,inf,0.49001347,0.5,1
-meas_sd,4,y6,-,-inf,inf,0.48723789,0.5,1
-meas_sd,5,y1,-,-inf,inf,0.51551926,0.5,1
-meas_sd,5,y2,-,-inf,inf,0.52331776,0.5,1
-meas_sd,5,y3,-,-inf,inf,0.48326815,0.5,1
-meas_sd,5,y4,-,-inf,inf,0.47910757,0.5,1
-meas_sd,5,y5,-,-inf,inf,0.50327233,0.5,1
-meas_sd,5,y6,-,-inf,inf,0.49705186,0.5,1
-meas_sd,6,y1,-,-inf,inf,0.52223919,0.5,1
-meas_sd,6,y2,-,-inf,inf,0.48641122,0.5,1
-meas_sd,6,y3,-,-inf,inf,0.47597189,0.5,1
-meas_sd,6,y4,-,-inf,inf,0.52595048,0.5,1
-meas_sd,6,y5,-,-inf,inf,0.51187305,0.5,1
-meas_sd,6,y6,-,-inf,inf,0.52425668,0.5,1
-meas_sd,7,y1,-,-inf,inf,0.52163477,0.5,1
-meas_sd,7,y2,-,-inf,inf,0.52112353,0.5,1
-meas_sd,7,y3,-,-inf,inf,0.47545353,0.5,1
-meas_sd,7,y4,-,-inf,inf,0.5150197,0.5,1
-meas_sd,7,y5,-,-inf,inf,0.48993218,0.5,1
-meas_sd,7,y6,-,-inf,inf,0.52777721,0.5,1
-meas_sd,7,Q1,-,-inf,inf,1.03957418,1,1
-shock_variance,0,fac1,-,-inf,inf,0.1036429,0.1,1
-shock_variance,0,fac2,-,-inf,inf,0.09803564,0.1,1
-initial_mean,0,mixture_0,fac1,-inf,inf,0,0,0
-initial_mean,0,mixture_0,fac2,-inf,inf,0,0,0
-initial_mean,0,mixture_0,fac3,-inf,inf,0,0,0
-initial_cov,0,mixture_0,fac1-fac1,-inf,inf,0.1764729,0.2,1
-initial_cov,0,mixture_0,fac2-fac1,-inf,inf,0.00524114,0,0
-initial_cov,0,mixture_0,fac2-fac2,-inf,inf,0.18362641,0.2,1
-initial_cov,0,mixture_0,fac3-fac1,-inf,inf,0.0056654,0,0
-initial_cov,0,mixture_0,fac3-fac2,-inf,inf,-0.00067522,0,0
-initial_cov,0,mixture_0,fac3-fac3,-inf,inf,0.23194739,0.2,1
+loading,0,y1,constant,-inf,inf,1.00161847,1.0,0.0
```

Comment:
Here you rename "delta" to "loading" instead of "control_coeff". This is wrong!

Reply:
done